src/DotNet.Consolidate/Models/OutputFormat.cs

```diff
@@ -0,0 +1,8 @@
+namespace DotNet.Consolidate.Models
+{
+    public enum OutputFormat
+    {
+        Text = 0,
```

Comment:
Yes, I know how it works. Why do we need this here?

Reply:
Defensive coding. In case someone comes in and adds an enum value and puts in different order the default doesn't change.

Reply:
Well, in my opinion it make sense only when you use compare enum values to integers, serialize them as integers or use them as flags.
But this doesn't make sense when you use enums as strings, we don't depend on underlying values at all.

Reply:
No I use it to say when i declare a variable as the enum that it defaults to Text.  I know the option attribute is setting default but I like the variable to start with the default when I do enums so that there isn't a case of it switching on us because someone did 
```c#
enum OutputFormat
{
    AFormat,
    Text,
    Json
}
```
Technically I should have sorted values.  It is something I do.  If I know a good default of an enum I will set it in enum so that it always starts with that value and is clear that this is the default so that it is not dependent on the order of the values to define the default and it switch because someone put in Aformat in wrong spot.

Reply:
```patch
diff --git a/src/DotNet.Consolidate/Models/OutputFormat.cs b/src/DotNet.Consolidate/Models/OutputFormat.cs
index 32fbf60..c502dd6 100644
--- a/src/DotNet.Consolidate/Models/OutputFormat.cs
+++ b/src/DotNet.Consolidate/Models/OutputFormat.cs
@@ -2,7 +2,7 @@ namespace DotNet.Consolidate.Models
 {
     public enum OutputFormat
     {
-        Text = 0,
-        Json
+        Json,
+        Text = 0
     }
 }
See the problem i fix by making sure default is specified?  Adding 0 just says this guy is the default value instead of it changing because of order of values and you can still sort the values.